eq-author/src/App/page/Logic/Routing/DestinationSelector/RoutingDestinationContentPicker.js

```diff
@@ -118,6 +118,7 @@ export const UnwrappedRoutingDestinationContentPicker = ({
         data-test="content-picker-select"
         onClick={() => setPickerOpen(true)}
         disabled={loading}
+        hasError={otherProps.hasError}
```

Comment:
Because `otherProps` is spread beneath this, it gets overwritten with `hasError` from there